tools/base/fpgainfo/board.c

```diff
@@ -386,7 +389,10 @@ fpga_result phy_command(fpga_token *tokens, int num_tokens, int argc,
 
 		fpgainfo_board_info(tokens[i]);
 		fpgainfo_print_common("//****** PHY ******//", props);
-		phy_group_info(tokens[i]);
+		res = phy_group_info(tokens[i]);
+		if (res != FPGA_OK) {
+			printf("phy group info is not supported\n");
```

Comment:
Use OPAE_ERR

Reply:
RC platform doesn't support MAC and PHY commends.  Prints not supported if user pass comments to fpgainfo .